src/Controls/src/Core/BindablePropertyConverter.cs

```diff
@@ -9,7 +11,7 @@ namespace Microsoft.Maui.Controls
 {
 	[Xaml.ProvideCompiled("Microsoft.Maui.Controls.XamlC.BindablePropertyConverter")]
 	[Xaml.TypeConversion(typeof(BindableProperty))]
-	public sealed class BindablePropertyConverter : TypeConverter, IExtendedTypeConverter
+	public sealed class BindablePropertyConverter : StringTypeConverterBase, IExtendedTypeConverter
```

Comment:
as ITypeDescriptorContext is a IServiceProvider, we probably could get rid of IExtendedTypeConverter (that adds a IServiceProvider parameter)